sklearn/datasets/descr/diabetes.rst

```diff
@@ -21,11 +21,11 @@ quantitative measure of disease progression one year after baseline.
       - sex
       - bmi     body mass index
       - bp      average blood pressure
-      - s1      tc, T-Cells (a type of white blood cells)
+      - s1      tc, total serum cholesterol
       - s2      ldl, low-density lipoproteins
       - s3      hdl, high-density lipoproteins
       - s4      tch, thyroid stimulating hormone
-      - s5      ltg, lamotrigine
+      - s5      ltg, serum concentration of lamotrigine
```

Comment:
ltg is almost certainly not anything to do with lamotrigine, an epilepsy drug.  In communication with one of the original authors, Hastie, he and Efron indicated that it's probably log of triglycerides.  That's a relevant quantity, although log_10 (which Efron suggested) gives unreasonable overall values (assuming the usual units).  Natural log is more plausible.  I'd suggest "ltg, possibly log serum triglycerides level".  See my original comment in issue #18940

Reply:
Updated.